pkgs/development/python-modules/linear-garage-door/default.nix

```diff
@@ -0,0 +1,54 @@
+{ lib
+, aiohttp
+, buildPythonPackage
+, dnspython
+, fetchFromGitHub
+, poetry-core
+, pythonOlder
+, tenacity
+}:
+
+buildPythonPackage rec {
+  pname = "linear-garage-door";
+  version = "0.2.9";
+  pyproject = true;
+
+  disabled = pythonOlder "3.8";
+
+  src = fetchFromGitHub {
+    owner = "IceBotYT";
+    repo = "linear-garage-door";
+    rev = "refs/tags/${version}";
+    hash = "sha256-hWWJgZnEItYaSxka7zBHPeVlTEiOqRnA2vg6/MvpJGE=";
+  };
+
+  postPatch = ''
+    sed -i "/--cov/d" pyproject.toml
+    sed -i "/--no-cov/d" pyproject.toml
+  '';
+
+  nativeBuildInputs = [
+    poetry-core
+  ];
+
+  propagatedBuildInputs = [
+    aiohttp
+    dnspython
+    tenacity
+  ];
+
+  # Module doesn't have tests
```

Comment:
yeah, you can't call this tests https://github.com/IceBotYT/linear-garage-door/tree/main/tests

Reply:
As the boilerplate is there, I guess that with one of the next releases there will be tests.